dts/common/nordic/nrf54h20.dtsi

```diff
@@ -987,6 +987,19 @@
 				status = "disabled";
 				interrupts = <470 NRF_DEFAULT_IRQ_PRIORITY>;
 			};
+
+			exmif: spi@95000 {
+				compatible = "nordic,nrf-exmif", "snps,designware-spi";
+				#address-cells = <1>;
+				#size-cells = <0>;
+				reg = <0x95000 0x500 0x95500 0xb00>;
+				reg-names = "wrapper", "core";
+				interrupts = <149 NRF_DEFAULT_IRQ_PRIORITY>;
+				clock-frequency = <DT_FREQ_M(400)>;
```

Comment:
what is the clock source here? looks like this should be referencing a clock?

Reply:
The clock source frequency is fixed and the application domain will only be able to turn it on or off. Currently there is no service that allows turning it on or off, but it is enabled by default.

Reply:
I mean that from a DT perspective, looks like there should be a clock that runs at 400MHz defined somewhere, unless this peripheral has a contained 400MHz clock

Reply:
![image](https://github.com/zephyrproject-rtos/zephyr/assets/25011557/cdc0dbfb-fddf-443e-bec2-dcc1fd55df94)

Reply:
so it looks like you should define a `fixed-clock` from an application perspective (similar to fll16m), as it's manages by sysctrl, then reference it as `clocks = <&hsfll_exmif>;`

![image](https://github.com/zephyrproject-rtos/zephyr/assets/25011557/79cf411a-e6f7-4ecc-8d1c-a942ee076786)

Reply:
Added the clock source.